src/Symfony/Component/Validator/Constraints/Image.php

```diff
@@ -184,5 +184,9 @@ public function __construct(
         $this->allowLandscapeMessage = $allowLandscapeMessage ?? $this->allowLandscapeMessage;
         $this->allowPortraitMessage = $allowPortraitMessage ?? $this->allowPortraitMessage;
         $this->corruptedMessage = $corruptedMessage ?? $this->corruptedMessage;
+
+        if (!\in_array('image/*', (array) $this->mimeTypes, true) && !\array_key_exists('mimeTypesMessage', $options ?? []) && null === $mimeTypesMessage) {
```

Comment:
a test case is missing with `image/*`

Reply:
See https://github.com/symfony/symfony/pull/45511